### ISSUE TITLE
SoftF1Loss miscalculates f1 score

### ISSUE THREAD

**Issue by panagiotidi** [kozistr/pytorch_optimizer#456]
## Describe the bug

`SoftF1Loss` returns `1.0 - f1.mean()` but it should simply return `f1`. 

## To Reproduce

`print(SoftF1Loss()(tensor([1.0, 0.0]), tensor([1.0, 0.0])))`

## Expected behavior

Should return 1, but returns 0.

**Comment by kozistr** [comment]
@panagiotidi hi! `SoftF1Loss()` is intended to be used as a loss function, designed for minimizing the F-beta loss (== maximizing the f-beta score), so `1.0 - f1.mean()` would be the correct form!

if you have any questions or feedback, feel free to leave a comment here :) thanks!

**Comment by panagiotidi** [comment]
Apologies, I was trying to use it as a metric function, I should have realized from its name it's a loss function.
